app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -64,7 +64,7 @@ export class TitusRunJobStageConfig extends React.Component<IStageConfigProps, I
       application: application.name,
       containerAttributes: {},
       env: {},
-      iamProfile: defaultIamProfile,
+      defaultIamProfile,
```

Comment:
this value will be sent down in the deploy config, and it's not a valid field, right?

Reply:
This is appended to the existing stage object. Let's say a stage has the iamProfile, we don't need to set it here; we use it directly. Let's say the value is not set, then we set the value from defaultIamProfile or set this as a placeholder. Is that what you are asking?

Reply:
Yeah, I don't think we should set `defaultIamProfile` on the cluster object itself, since that object is what gets sent in the api call to gate.

Reply:
Thanks for catching it! Will send out a fix shortly.